pkg/apis/submariner.io/v1/types.go

```diff
@@ -46,6 +46,7 @@ type EndpointSpec struct {
 	ClusterID     string            `json:"cluster_id"`
 	CableName     string            `json:"cable_name"`
 	Hostname      string            `json:"hostname"`
+	Hostname2     string            `json:"hostname2"`
```

Comment:
I think these type changes shouldn't be a part of this commit?

Reply:
I imagine the point here was to prove that the test works ;-). (But yes, they’ll need to be removed before the merge.)